### PR TITLE
Jake branch

### PR DESCRIPTION
**Pull request by Fishervjacob**

### CODE REVIEW

File: src/main/java/ludum/dare/trait/HealthTrait.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package ludum.dare.trait;
+
+/**
+ * Created by jake on 8/22/2015.
+ */
+public class HealthTrait extends Trait {
+    public int Health;
```
Comment (MondayHopscotch) [inline src/main/java/ludum/dare/trait/HealthTrait.java]: WOw. just wow.

Comment (Kenoshen) [inline src/main/java/ludum/dare/trait/HealthTrait.java]: You are a horrible person...

File: src/main/java/ludum/dare/trait/HealthTrait.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    public HealthTrait(GameObject obj) {
+        super(obj);
+    }
+
+    @Override
+    public Class[] requires() {
+        return new Class[0];
+    }
+    public void setHealth(int h){
```
Comment (Kenoshen) [inline src/main/java/ludum/dare/trait/HealthTrait.java:16]: don't need this setHealth thing

Comment (MondayHopscotch) [inline src/main/java/ludum/dare/trait/HealthTrait.java:16]: I would opt for methods like damage(int damage) and heal(int amount) and make health private.

From there we can have other methods like isDead() and such

**Comment by Kenoshen** [comment]
Talked about re-org, looks good otherwise

**Pull request merged**